consensus/state.go

```diff
@@ -1486,6 +1486,19 @@ func (cs *State) recordMetrics(height int64, block *types.Block) {
 
 		for i, val := range cs.LastValidators.Validators {
 			commitSig := block.LastCommit.Signatures[i]
+			privValAddress := cs.privValidator.GetPubKey().Address()
+			if cs.privValidator != nil && bytes.Equal(val.Address, privValAddress) {
+				label := []string{
+					"validator_address", privValAddress.String(),
+				}
+				cs.metrics.ValidatorPower.With(label...).Set(float64(val.VotingPower))
+				if !commitSig.Absent() {
```

Comment:
This checks for BlockIDFlagAbsent. Wouldn't BlockIDFlagNil (nil vote) also count as a miss?

Reply:
https://github.com/tendermint/tendermint/pull/4283 did the follow up here